cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -203,32 +201,42 @@ func (o *objectGraph) getDiscoveryTypes() ([]metav1.TypeMeta, error) {
 	return discoveredTypes, nil
 }
 
+func getCRDList(proxy Proxy, crdList *apiextensionsv1.CustomResourceDefinitionList) error {
+	c, err := proxy.NewClient()
+	if err != nil {
+		return err
+	}
+
+	if err := c.List(ctx, crdList, client.MatchingLabels{clusterctlv1.ClusterctlLabelName: ""}); err != nil {
```

Comment:
You can also use `client.HasLabels` if you don't care about the value part of the label

Reply:
done